packages/cli/src/cmds/config.ts

```diff
@@ -4,37 +4,118 @@
 "use strict";
 
 import { Argv } from "yargs";
+import * as path from "path";
 import { YargsCommand } from "../yargsCommand";
-import { FxError, Result, ok, LogLevel } from "@microsoft/teamsfx-api";
+import { FxError, Question, Result, ok, err, LogLevel } from "@microsoft/teamsfx-api";
 import { UserSettings, CliConfigOptions, CliConfigTelemetry } from "../userSetttings";
 import CLILogProvider from "../commonlib/log";
+import {
+  readProjectSecrets,
+  writeSecretToFile,
+  getSystemInputs,
+  readConfigs,
+  toYargsOptions,
+} from "../utils";
+import CliTelemetry from "../telemetry/cliTelemetry";
+import {
+  TelemetryEvent,
+  TelemetryProperty,
+  TelemetrySuccess,
+} from "../telemetry/cliTelemetryEvents";
+import activate from "../activate";
+import { NonTeamsFxProjectFolder, ConfigNameNotFound } from "../error";
+import * as constants from "../constants";
+
+const CryptoDataMatchers = new Set([
```

Comment:
Are these also used in the vscode? If yes, I think you can maintain these in the high level.

Reply:
This is a good point, I updated codes to reuse the exported function in fx-core. VS Code doesn't share the same way because it uses regex to check now.